handout/forms/template-driven/validating_forms.md

```diff
@@ -15,7 +15,7 @@ Using the template-driven approach, form validation is a matter of following HTM
 <input type="text" ... pattern=".{3,8}" required>
 
 <!-- alphanumeric field of specific length -->
-<input type="text" pattern="[A-Za-z0-9]{0,5}">
+<input type="text" ... pattern="[A-Za-z0-9]{0,5}">
```

Comment:
I was just matching the code examples above for consistency. I agree that the ellipsis feels a bit awkward. I can remove them if desired.